charts/spiderpool/templates/role.yaml

```diff
@@ -152,27 +312,70 @@ rules:
 - apiGroups:
   - spiderpool.spidernet.io
   resources:
-  - spidercoordinators
+  - spidercoordinators/status
+  - spiderippools/status
+  - spidersubnets/status
+  verbs:
+  - get
+  - patch
+  - update
+---
+{{- if or .Values.ipam.enableIPv4 .Values.ipam.enableIPv6 }}
+{{- if or .Values.clusterDefaultPool.installIPv4IPPool .Values.clusterDefaultPool.installIPv6IPPool .Values.coordinator.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.spiderpoolInit.name | trunc 63 | trimSuffix "-" }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - endpoints
+  - namespaces
+  - pods
+  - pods/status
   verbs:
-  - create
   - delete
+  - deletecollection
   - get
   - list
   - patch
   - update
   - watch
 - apiGroups:
-  - spiderpool.spidernet.io
+  - ""
   resources:
-  - spidercoordinators/status
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
   verbs:
+  - create
+  - delete
   - get
+  - list
   - patch
   - update
+  - watch
 - apiGroups:
```

Comment:
Would it be possible to remove this rule? This rule include `get/list` verb of `secrets` at cluster-wide, which may cause some security risks. @cyclinder

Reply:
Also same question exists in Spiderpool controller.

Reply:
Yes, You can boldly delete them. CI will tell us if we cannot get/list/watch some resources.

Reply:
OK! Let me give it a try